app/projects/ideas/page.tsx

```
@@ -4,22 +1,460 @@
+        let foundTheKey = null;
+        for (const key of keys) {
+          if (key.toUpperCase() === "THE") {
+            // So sánh không phân biệt hoa thường để tìm
+            foundTheKey = key;
+            break;
+          }
+        }
+        if (foundTheKey) {
+          console.log(
+            `DICTIONARY_DEBUG: Found 'THE' as key: '${foundTheKey}'. Value:`,
+            (dictionary as any)[foundTheKey]
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)